human-forecasts/redeploy-forecast-app.R

```diff
@@ -19,8 +27,6 @@ if (weekdays(Sys.Date()) != "Monday") {
 
 saveRDS(submission_date, "human-forecasts/data/submission_date.RDS")
```

Comment:
RDS?

Reply:
doing that to pass the submission date into the forecast app

Reply:
it should be `.rds`